client/tripPlan/tripOverview/travelFriendList.js

```diff
@@ -2,26 +2,90 @@ import React from 'react';
 import { connect } from 'react-redux';
 import TravelFriendItem from './travelFriendItem';
 import List from 'material-ui/lib/lists/list';
+import ListItem from 'material-ui/lib/lists/list-item';
+import Avatar from 'material-ui/lib/avatar';
+import DropDownMenu from 'material-ui/lib/DropDownMenu/DropDownMenu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+import FriendItem from '../../friend/friendItem';
+import { toggleList } from './travelFriendActions';
+import { addFriends } from './travelFriendActions';
+import { toggleAddFriend } from '../../friend/friendActions';
 
 const mapStateToProps = state => ({
   travelFriends: state.tripPlan.tripOverview.get('travelFriends'),
+  trip: state.tripPlan.selectedTrip,
+  friends: state.friend.get('friends'),
+  addedFriends: state.friend.get('friends').filter(friend => friend.addedToTrip),
+  listToggle: state.tripPlan.tripOverview.get('showList'),
 });
 
-let TravelFriendList = ({ travelFriends }) => (
+const mapDispatchToProps = dispatch => ({
+  onClickToggleList: () => dispatch(toggleList()),
+  onFriendClick: id => dispatch(toggleAddFriend(id)),
+  onClickAddFriends: (friends, trip) => dispatch(addFriends(friends, trip)),
+});
+
+let TravelFriendList = ({ travelFriends, friends, listToggle, onClickToggleList, onFriendClick, addedFriends, trip, onClickAddFriends }) => ( // eslint-disable-line
   <div>
+
     <List>
       {travelFriends.map(friend =>
         <TravelFriendItem key={ friend.id } {...friend} />
       )}
     </List>
+
+    <ListItem primaryText={listToggle ? 'Hide list' : 'Add more friends!'}
+      leftAvatar={
+        <Avatar src={listToggle ?
+          'http://www.aisletracker.com/afc/images/icons/red_minus.png' : '../../assets/add.png'}
```

Comment:
can we download this image and serve from assets rather than fetching from the web? Or just use [Material Design icons](https://design.google.com/icons/#ic_add) (these come packaged with Material-UI).